packages/react-storage/src/components/StorageImage/StorageImage.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import { Image, ComponentClassNames } from '@aws-amplify/ui-react';
+import { useStorageURL } from '@aws-amplify/ui-react/internal';
+
+import type { StorageImageProps } from './types';
+
+export const StorageImage = ({
+  imgKey,
+  accessLevel,
+  identityId,
+  fallbackSrc,
+  onStorageError,
```

Comment:
Why `onStorageError` instead of `onError`?

Reply:
I believe onStorageError is if the Storage.get command fails somehow, and onError is the HTML onError event for an image element?

Reply:
Got it. Is `onStorageError` too generic since it refers to `Storage` directly? Not sure if there will be additions to this component's  interactions with `Storage` in the future, but if so might be kind of having `onStorageError` and adding a more specific namespaces for other `Storage` error handlers. Maybe `onStorageGetError` or `onGetError`?

Reply:
I am open to any idea about the naming here. `onStorageGetError` sounds good to me.

Reply:
+1 for `onStorageGetError` since it's more specific.